UM/Qt/qml/UM/Slider.qml

```diff
@@ -0,0 +1,71 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.15
+
+import UM 1.7 as UM
+
+Slider
+{
+    id: slider
+    //Draw line
+    background: Rectangle
+    {
+        id: backgroundLine
+        height: UM.Theme.getSize("print_setup_slider_groove").height
+        width: parent.width - UM.Theme.getSize("print_setup_slider_handle").width
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.verticalCenter: parent.verticalCenter
+        color: UM.Theme.getColor("lining")
+
+        Repeater
+        {
+            id: repeater
+            anchors.fill: parent
+            model: 11
```

Comment:
What is `11` and will it always be `11` for a generically named component named `Slider`

Reply:
This is some QML magic. It creates 11 empty model items. These are used to generate the dots in the background of the slider, 1 for each model. 

I tried using an alias or property for this value but it does not work for some reason. If we want more/less ticks in the future I think we can figure it out then.

Reply:
I was mistaken. I added an alias for this value at the top of the file